source/_patterns/02-molecules/blocks/step/step.twig

```diff
@@ -1,8 +1,19 @@
-{% set url_attributes = url_attributes ? url_attributes : create_attribute() %}
```

Comment:
AFAIK, this one will not actually have a URL, but instead will use the `read-more.twig` toggle.   I've implemented that here, allowed the tag to be customized, and added a few if statements.